static/app/views/settings/organizationAuthTokens/authTokenRow.tsx

```diff
@@ -109,11 +111,15 @@ export function OrganizationAuthTokensAuthTokenRow({
       </div>
 
       <LastUsedDate>
-        <LastUsed
-          dateLastUsed={token.dateLastUsed}
-          projectLastUsed={projectLastUsed}
-          organization={organization}
-        />
+        {isProjectLoading ? (
+          <LoadingIndicator mini />
```

Comment:
Should we use a Placeholder here instead?

Reply:
Hmm not sure what a good placeholder would be 🤔 there was kind of a placeholder before - it would show:

```
a few seconds ago
```

and after an instant, when the projects finished loading, be replaced with `a few seconds ago in project xxx`.

Here two videos to compare:


https://github.com/getsentry/sentry/assets/2411343/a061da2a-f1de-4cbb-baeb-ad5a8e02d09d

vs

https://github.com/getsentry/sentry/assets/2411343/9198d42b-db77-4677-8be7-2c5404965d1a

(it's subtle and fast here because this is local, obviously 😅 )

I'm also happy to leave this as it was if you think that's better, no strong feelings, just felt slightly off while working on this to me :)

Reply:
We have a component called `Placeholder` that just renders a gray box. We use it commonly for list that are loading

![image](https://github.com/getsentry/sentry/assets/1421724/5d618388-b442-4673-a127-e8231f49029a)

Reply:
Ahhh TIL, now that makes sense of course. I updated this accordingly!